flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableSourceStatisticsReportTest.java

```diff
@@ -255,7 +257,7 @@ protected Map<String, List<Object>> getDataMap() {
     }
 
     private static void assertHiveTableOrcFormatTableStatsEquals(
-            TableStats tableStats, int expectedRowCount, long nullCount) {
+            TableStats tableStats, String hiveVersion, int expectedRowCount, long nullCount) {
```

Comment:
To get `hiveVersion`, you can use `String hiveVersion = HiveShimLoader.getHiveVersion();` directly. So that, we won't need to pass the `hiveVersion`.

Reply:
> To get `hiveVersion`, you can use `String hiveVersion = HiveShimLoader.getHiveVersion();` directly. So that, we won't need to pass the `hiveVersion`.

Done!